node_package/tests/ReactOnRails.test.js

```diff
@@ -21,7 +21,7 @@ describe('ReactOnRails', () => {
 
     document.body.innerHTML = '<div id="root"></div>';
     // eslint-disable-next-line no-underscore-dangle
-    const actual = ReactOnRails.render('R1', {}, 'root')._reactInternalFiber.type;
+    const actual = ReactOnRails.render('R1', {}, 'root')._reactInternals.type;
```

Comment:
This is internal logic in React, and there is no documentation about it. But in React 17, `ReactOnRails.render('R1', {}, 'root')` only has `._reactInternals` In React 18, this is changed again.